tests/cases/conformance/classes/propertyMemberDeclarations/initializationOrdering1.ts

```diff
@@ -0,0 +1,18 @@
+// @target: esnext, es2021, es2022
+// @useDefineForClassFields: true

```

Comment:
```suggestion
// @target: esnext, es2021, es2022
// @useDefineForClassFields: false, true
```

This should work and let you eliminate that other file, and make the baselines more clear, I think.